doc/modules/partial_dependence.rst

```diff
@@ -57,38 +81,68 @@ and a two-way PDP between the two features::
     >>> features = [0, 1, (0, 1)]
     >>> plot_partial_dependence(clf, X, features) #doctest: +SKIP
 
+Individual conditional expectation plots for the features ``0`` and ``1`` can
+be generated with::
+
+    >>> from sklearn.inspection import plot_individual_conditional_expectation
+    >>> features = [0, 1]
+    >>> plot_individual_conditional_expectation(clf, X, features) #doctest: +SKIP
+
 You can access the newly created figure and Axes objects using ``plt.gcf()``
 and ``plt.gca()``.
 
 For multi-class classification, you need to set the class label for which
-the PDPs should be created via the ``target`` argument::
+the PDP/ICEs should be created via the ``target`` argument::
 
     >>> from sklearn.datasets import load_iris
     >>> iris = load_iris()
     >>> mc_clf = GradientBoostingClassifier(n_estimators=10,
     ...     max_depth=1).fit(iris.data, iris.target)
     >>> features = [3, 2, (3, 2)]
     >>> plot_partial_dependence(mc_clf, X, features, target=0) #doctest: +SKIP
+    >>> features = [3, 2]
+    >>> plot_individual_conditional_expectation(mc_clf, X, features, target=0) #doctest: +SKIP
 
 The same parameter ``target`` is used to specify the target in multi-output
 regression settings.
 
-If you need the raw values of the partial dependence function rather than
-the plots, you can use the
-:func:`sklearn.inspection.partial_dependence` function::
+If you need the raw values of the partial dependence/individual conditional
+expectation function rather than the plots, you can use functions
+:func:`sklearn.inspection.partial_dependence` and
+:func:`individual_conditional_expectation`::
 
     >>> from sklearn.inspection import partial_dependence
+    >>> from sklearn.inspection import individual_conditional_expectation
 
     >>> pdp, axes = partial_dependence(clf, X, [0])
     >>> pdp
     array([[ 2.466...,  2.466..., ...
     >>> axes
     [array([-1.624..., -1.592..., ...
 
-The values at which the partial dependence should be evaluated are directly
-generated from ``X``. For 2-way partial dependence, a 2D-grid of values is
-generated. The ``values`` field returned by
-:func:`sklearn.inspection.partial_dependence` gives the actual values
+    >>> ice, axes = individual_conditional_expectation(clf, X, [0])
+    >>> ice
+    array([[[0.9983247 , 0.9983247 , 0.9983247 , ..., 0.99461002,
+             0.99890308, 0.99890308],
+            [0.23127816, 0.23127816, 0.23127816, ..., 0.08522466,
+             0.31495788, 0.31495788],
+            [0.99836689, 0.99836689, 0.99836689, ..., 0.99474525,
+             0.99893072, 0.99893072],
+            ...,
+            [0.18632824, 0.18632824, 0.18632824, ..., 0.06621577,
+             0.25922884, 0.25922884],
+            [0.92358666, 0.92358666, 0.92358666, ..., 0.78915304,
+             0.94864021, 0.94864021],
+            [0.99704493, 0.99704493, 0.99704493, ..., 0.99051952,
+             0.99806429, 0.99806429]]])
+    >>> axes
+    [array([-1.624..., -1.592..., ...
```

Comment:
I think it's OK not to write the example here, considering that you properly mention the existence of the function before, and its usage is similar to `partial_dependence` which is illustrated. (Also, I don't expect these functions to be really used by anyone)

Reply:
Removed in https://github.com/scikit-learn/scikit-learn/pull/16164/commits/61467e80861895176a8fc0533c4e79de241bbbea